ETC Course: 28. What Are Privacy Coins? (#1246)
ethereumclassic/ethereumclassic.github.io: content/blog/2023-06-29-ethereum-classic-course-28-what-are-privacy-coins/index.md

In the next two classes we will cover these other topics:

- Class 29: What Are Decentralized Finance (DeFi) Coins?
- Class 30: What Are File Storage Coins?

## Default Privacy in Standard Blockchains

Blockchains such as Ethereum Classic (ETC) and Bitcoin (BTC) are very transparent in the sense that all transactions, smart contracts, balances, and even the code of the software clients are public information, so anyone may audit them and check if everything is working as designed. 

Privacy in these systems is given by the fact that users may control their addresses by just holding secret private keys, so their corresponding public addresses are not tied to their identities, persons, or entities. For this reason, if users are diligent in keeping the privacy of their movements on the blockchain, then a certain level of privacy is possible.

Nevertheless, with advances in data science, the capture of information from many sources, and triangling economic transactions, it is becoming easier and more common for external observers to figure out who owns what inside these public and transparent systems.

## What Are Privacy Coins?

![Privacy.](./1.png)

> In the context described above, and to enhance individual user security, privacy coins have made the tradeoff of reducing transparency of their systems to increase the secrecy of transactions, addresses, and balances.

Reviewer: Liner 38: Rewrite for readability
```suggestion
In the above context, and in order to enhance the security of individual users, privacy coins have made the tradeoff of reducing the transparency of their systems in order to increase the secrecy of transactions, addresses, and balances.
```

There are several coins that have been launched to enhance privacy, some of which are frequently used and have a lot of transaction activity. 

The segment as a whole has reached a significant market capitalization. 

## Privacy Methods: Zero Knowledge

![Zcash uses zero knowledge proofs.](./2.png)

If there are two parties handling secret information and one sends the information to the other in a way that stays hidden, the receiver could modify or not the data even without knowing it, and then show it to the sender, asking them if the data was modified. Because the sender does know the information, they can easily answer the question correctly still without revealing the contents to the receiver.

After a large number of tries, if the sender perfectly answers all the questions, then it is astronomically likely that the information is true, even if the receiver does not know what is its content. This is the basis of “zero knowledge proofs”.

Zcash is the cryptocurrency that has implemented this method to create a privacy blockchain. It is the second largest privacy coin in the industry with a market capitalization of $405 million at the time of this writing. 

> On Zcash users may opt to send private or transparent transactions.

Reviewer: Line 56: Added comma
```suggestion
On Zcash, users may opt to send private or transparent transactions.
```

**Stealth addresses:** This technology hides the receiver of money in the network by creating “stealth addresses” for each receiver and for each transaction. 

**Ring signatures:** This type of signature mixes the signatures of several senders in each transaction so it is impossible to know who is the sender.

**Ring Confidential Transactions:** This component of the transaction uses cryptography to hide the amounts sent in each transfer of coins in the system.

These three technologies combined provide a large degree of privacy to users. On Monero, all transactions are hidden by default and all use the three methods mentioned above.

## Other Privacy Methods

![Top ten privacy coins on CoinMarketCap.](./4.png)

There are several other privacy techniques in the industry, such as mixers and Mimblewimble. 

The industry has over 70 networks offering privacy using the techniques mentioned in this post and combinations thereof.

The combined market capitalization of this segment of the industry is $5.3 billion at the time of this writing.

> In the image in this section you may see the top ten privacy coins on CoinMarketCap.

Reviewer: Line 62: Rewrite for readability
```suggestion
The accompanying image shows the top ten privacy coins listed on CoinMarketCap.
```

## Risks of Privacy Coins

> A general risk for these coins, that is usually pointed out, is that the transparency of the traditional blockchains makes them easy to audit and reveal bugs or flaws that can subsequently be fixed, but if privacy coins hide all or part of the information, then it is impossible to audit their design, function, monetary supply, and other key metrics.

Reviewer: Line 86:  Rewrite for readability
```suggestion
A commonly known general risk for privacy coins is that the transparency of the traditional blockchains makes them easy to audit, revealing bugs or flaws that can subsequently be fixed, but if privacy coins hide all or part of that information, it becomes impossible to audit the design, function, monetary supply, or other key metrics of the blockchain.
```
